.github/stale.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before an stale issue is closed
+daysUntilClose: 30
+
+# Label to use when marking an issue as stale
+staleLabel: triage/stale
+
+# Comment to post when marking an issue as stale.
```

Comment:
Since we're using `pulls` in line 21, I'd recommend moving the following section under `issues`:
```yaml
issues:
  markComment: |-
    This issue has been automatically marked as stale because it has not had
    recent activity. It will be closed if no further activity occurs. Thank you
    for your contributions.
  # Comment to post when closing a stale issue.
  closeComment: |-
    This issue has been automatically closed because it has not had recent
    activity since being marked as stale.
```

Reply:
make sense here also 😄